### DISCUSSION TITLE
extends config:base overwrites prHourlyLimit

### DISCUSSION THREAD

**Discussion by irrandon** [renovatebot/renovate#10899] [Request Help] [answered]
Hello everyone :)
We updated renovate pro from 1.3.1 to 1.4.2. After that renovate ignores the prHourlyLimit in the config and change it to 2.
Is that a bug?

**Comment by rarkins** [comment]
You haven't given enough details

**Reply by irrandon** [reply]
we are using gitlab-ce 14.0.5
Our config:
https://gist.github.com/irrandon/e5da917d61578befb19e66de290703d8
log:
https://gist.github.com/irrandon/2d6d4502b8736aa56ddad7524dfd0d11

**Reply by rarkins** [reply]
Some of the removed log snippets prior to the resolution may be relevant, can you add them back?

**Reply by irrandon** [reply]
here is the full log (:
https://gist.github.com/irrandon/b617122e93db0592c88f8bad3c8c0266

**Reply by rarkins** [reply]
May take a while to look over that. In the meantime, can you try using the ignorePresets feature to ignore the :prHourlyLimit2 preset?

**Reply by irrandon** [reply]
Unfortunately that does not work. Same result.

**Comment by rarkins** [comment, marked as answer]
It would be better if you stop extending config:base and instead extend its sub presets which you need.

I think I know now why this is happening and it's not something we are likely to be able to change soon. In general we don't recommend the use of presets on global config like this and may deprecate it in future.

**Reply by irrandon** [reply]
That worked (: thank you for your help
